server/events/vcs/github_client_test.go

```diff
@@ -312,15 +312,21 @@ func TestGithubClient_MergePull(t *testing.T) {
 		},
 	}
 
+	// Use a real GitHub json response and edit the mergeable_state field.
+	jsBytes, _ := ioutil.ReadFile("fixtures/github-pull-request-baserepo.json")
```

Comment:
You still need to write tests that edit the mergeable_state field and check that we use the correct merge options in our request right?

Reply:
Yeah, good point, I'll do that tomorrow.

Reply:
@brndnmtthws I'm going to carry this patch for an urgent release so I'll handle the tests myself